pkgs/by-name/fo/forgejo-runner/package.nix

```diff
@@ -64,21 +68,27 @@ buildGoModule rec {
   ldflags = [
     "-s"
     "-w"
-    "-X code.forgejo.org/forgejo/runner/v11/internal/pkg/ver.version=${src.rev}"
+    "-X code.forgejo.org/forgejo/runner/v12/internal/pkg/ver.version=${src.rev}"
   ];
 
   checkFlags = [
     "-skip ${lib.concatStringsSep "|" disabledTests}"
   ];
 
   postInstall = ''
-    # fix up go-specific executable naming derived from package name, upstream
+    # Fix up go-specific executable naming derived from package name, upstream
     # also calls it `forgejo-runner`
     mv $out/bin/runner $out/bin/forgejo-runner
+
+    # provide backward compatbility since v12 removed bundled git
```

Comment:
Nit: There has never been a bundled git. It's always been `go-git`. What's new is that in addition to `go-git`, `git` is now required for certain codepaths.